variants/arduino_101/variant.cpp

```diff
@@ -240,7 +240,12 @@ void initVariant( void )
     variantGpioInit();
     variantPwmInit();
     variantAdcInit();
-
+    
+    //set RTC clock divider to 32768
+    *SYS_CLK_CTL |= 0x00000078;
```

Comment:
Use some #defines with human-readable names, please. I'd rather not have to look up the hardware
spec in order to learn what each line is doing. Same goes for all the magic numbers here.